test/lib/expr_test_utils.cc

```diff
@@ -389,6 +389,121 @@ std::pair<evaluation_inputs, std::unique_ptr<evaluation_inputs_data>> make_evalu
     return std::pair(std::move(inputs), std::move(data));
 }
 
+mock_database_impl::mock_database_impl(schema_ptr table_schema) {
+    _table_schema = table_schema;
+    _keyspace_metadata = data_dictionary::keyspace_metadata::new_keyspace(
```

Comment:
Please use the regular constructor member initializer list style, it makes it easier to add things later if they don't have default constructors.